api/schema/RideSchema.js

```diff
@@ -2,16 +2,20 @@ import nodemailer from 'nodemailer'
 import { UserTC, RideTC, LocationTC, User, Ride, Location } from '../models'
 import { isRideFull } from '../utils/rideUtils'
 
-const transporter = nodemailer.createTransport({
-  host: 'smtp.gmail.com',
-  port: 465,
-  secure: true,
-  auth: {
-    // TODO: Use OAuth instead 😳
-    user: process.env.SENDER_EMAIL,
-    pass: process.env.SENDER_EMAIL_PASSWORD,
-  },
-})
+if (process.env.SENDER_EMAI && process.env.SENDER_EMAIL_PASSWORD) {
```

Comment:
Safety checks for if environment variables set, currently in my OSA team we haven't been provided them yet so I've made the transporter optional for now